packages/apps-config/src/api/chain/index.ts

```diff
@@ -1,13 +1,13 @@
 // Copyright 2017-2021 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import Phala from './phala';
 import Cere from './cere';
+import Phala from './phala';
 
 // alphabetical, based on the actual displayed name
 export default {
-  'Phala PoC-3': Phala,
   'Cerebellum Network Testnet': Cere,
   'Cerebellum Network Testnet Beta': Cere, // For Substrate cluster
   'Local Testnet': Cere, // For Nodes bootstraped from --local chainspec
+  'Phala PoC-3': Phala
```

Comment:
Do not include changes to PR that are not related to its main purpose

Reply:
I had to run eslint --fix so that all checks pass

Reply:
Let's include them in a separate PR.